sda-commons-server-cloudevents/src/test/java/org/sdase/commons/server/cloudevents/app/consume/ContractCreatedEvent.java

```diff
@@ -0,0 +1,46 @@
+package org.sdase.commons.server.cloudevents.app.consume;
+
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import java.net.URI;
+import org.sdase.commons.server.cloudevents.CloudEventV1;
+
+public class ContractCreatedEvent extends CloudEventV1<ContractCreatedEvent.ContractCreated> {
+
+  @JsonPropertyDescription(
```

Comment:
You may use `@JsonSchemaDefault` here to keep the original description.

There is also `@JsonSchemaInject` which could be used to define a constant value:

```
@JsonSchemaInject(json = "{ \"const\": \"/SDA-SE/insurance-contract/insurance-contract-stack/insurance-contract-service\"}")
or 
@JsonSchemaInject(strings = @JsonSchemaString(path = "const", value = "/SDA-SE/insurance-contract/insurance-contract-stack/insurance-contract-service"))
```

By documentation, `@JsonSchemaInject` can also be used at type level (to avoid overriding), but that seems to be buggy or I don't know how to use it.

```
This at type level:
@JsonSchemaInject(strings = @JsonSchemaString(path = "/properties/source/const", value = "/SDA-SE/insurance-contract/insurance-contract-stack/insurance-contract-service"))

leads to this in the Api-Yaml:

  ContractCreatedEvent:
...
      ? ""
      : properties:
          source:
            const: "/SDA-SE/insurance-contract/insurance-contract-stack/insurance-contract-service"
...

And 
@JsonSchemaInject(strings = @JsonSchemaString(path = "properties/source/const", value = "/SDA-SE/insurance-contract/insurance-contract-stack/insurance-contract-service"))

is not affecting the result
```

Maybe worth to investigate further.

Reply:
I switched to `JsonSchemaDefault`. It's nice to also have the original description! 👍